master/internal/experiment.go

```diff
@@ -727,11 +727,13 @@ func (e *internalExperiment) handleContinueExperiment(reqID model.RequestID) (*i
 			e.syslog.Error(err)
 			return nil, true
 		}
-		if trial.State != model.CompletedState {
```

Comment:
this code feels a little complicated to me now

maybe something like a switch statement could make this more clear we have three cases going on

```
switch {
case errors.Is(err, sql.ErrNoRows):
    // Trial doesn't exist, don't do anything
case err != nil:
   e.updateState ... 
case err == nil:
   if state...
}
```

Reply:
Done!